pkg/kubelet/kubelet.go

```diff
@@ -2659,7 +2659,7 @@ func (kl *Kubelet) GetPodByName(namespace, name string) (*api.Pod, bool) {
 }
 
 func (kl *Kubelet) updateRuntimeUp() {
-	if _, err := kl.containerRuntime.Version(); err != nil {
+	if err := kl.containerRuntime.CheckStatus(); err != nil {
```

Comment:
@yifan-gu, is this a good health check for rkt?
I realized that `Version()` is a no-op for rkt since it checks the cached version